### ISSUE TITLE
Accessibility - Steps Screens - List of Links under "Topics" Not Read as Links

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#37]
For some reason the list of links under the Topic section of each of the Steps screens does not read what look like links as such.

My guess is they are not HTML A HREF created links but something else to act like a link. By having it this way screen reader users have no idea they are links either inside the application of to external websites.

This might be a similar issue with the bread crumbs not being read as links.

Please look at the bread crumb issues too for similar problem.

**Comment by adriennerlong** [comment]
I've flagged this as a high priority issue. We will ask the developers to address before 9/11.

**Comment by jfc3-dol** [comment]
@adriennerlong,

Thanks, for getting the developers to fix this item. It's important for those using screen readers.

**Comment by jfc3-dol** [comment]
After further looking it seems that VoiceOver is reading everything under the Topics and Further Resources as one item. When VoiceOver should read the Topic and arrow as one item and the each of the links are getting read as one continuous stream of text and not as individual links in this case.

This is true for all the other sections too. I'm looking at the looking at the Step 1 main screen.

**Comment by jfc3-dol** [comment]
@adriennerlong,

VoiceOver is now reading each link under the topic section as a link and a heading level three (h3). These should not be heading level three (h3) items.

**Comment by adriennerlong** [comment]
From ICF: Fixed

**Comment by jfc3-dol** [comment]
VoiceOver is reading the links under each Step's (1 -8) "Topics" sections unordered list items as a heading level three (h3) and a link. They should only be links and not be a heading level three (h3).

Please see Step 1 main screen as an example.

**Comment by sanganalakshay** [comment]
Fixed

**Comment by jfc3-dol** [comment]
The list of links under the "Topics" sections on the different Steps screens are only being read as links now.

Closing issue.
